genie-common/src/main/java/com/netflix/genie/common/util/Constants.java

```diff
@@ -0,0 +1,235 @@
+/*
+ *
+ *  Copyright 2016 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.common.util;
+
+/**
+ * A class holding some constants to be used everywhere.
+ *
+ * @author amsharma
+ * @author tgianos
+ * @since 3.0.0
+ */
+public final class Constants {
```

Comment:
Since most of these Constants are used in the core project would it make more sense to move this file there? Are any of them even used in Common or would be used by something like the client? Unlikely from the looks of the constants